Common/include/config_structure.hpp

```diff
@@ -1353,24 +1372,12 @@ class CConfig {
 	 * \return Value of the limits for the sections.
 	 */
 	su2double GetSection_Location(unsigned short val_var);
-
```

Comment:
Thanks a lot for your comments and, for taking the time to review the implementation. Code style (including indentation) is very important to keep the source code under control. I have fixed the indentation of this file but, maybe, before releasing SU2 the entire indentation of the software should be updated.